spec/src/main/asciidoc/architecture.adoc

```diff
@@ -32,6 +32,39 @@ This specification documents the component specification contents of each releas
 
 The specifications for each release are listed in reverse chronological order.
 
+[[microprofile7.0]]
+=== MicroProfile 7.0
+Based on MicroProfile's time-boxed release process, this is a major release that does include backward incompatible changes. This release no longer brings in Jakarta EE 10 Core Profile, instead it declares a loosely-coupled dependency on Jakarta EE 10 Core Profile. This release does not include MicroProfile Metrics as MicroProfile Telemetry 2.0 offers the Metrics capabilities in addition to Logs and Tracing. Therefore, MicroProfile Metrics became a standalone specification.
```

Comment:
> This release no longer brings in Jakarta EE 10 Core Profile, instead it declares a loosely-coupled dependency on Jakarta EE 10 Core Profile.

I'm not sure what the difference between these two statements is. Is there a mistake here? Otherwise I would find it helpful if there was more information on what is meant by a "loosely-coupled dependency".

Reply:
Previously, the BOM has Jakarta EE 10 Core Profile included but now it does not have it included.